src/Compute/Compute/Manual/PSVirtualMachinePatchAssessmentResult.cs

```diff
@@ -29,7 +29,7 @@ public class PSVirtualMachinePatchAssessmentResult
         public int? CriticalAndSecurityPatchCount { get; set; }
         public int? OtherPatchCount { get; set; }
         public DateTime? StartDateTime { get; set; }
-        public IList<VirtualMachineSoftwarePatchProperties> Patches { get; set; }
+        public IList<VirtualMachineSoftwarePatchProperties> AvailablePatches { get; private set; }
```

Comment:
So the issue was that the PS object made up a nonexistent Patches property instead of the AvailablePatches property which actually exists on the VirtualMachineAssessPatchesResult.cs object?